src/components/app-configure/gasless/AppGasTankForm.vue

```diff
@@ -11,6 +11,7 @@ const { gaslessChains, appChains } = chainManagementStore
 const { gastankList } = useGaslessStore()
 const emits = defineEmits(['close', 'submit'])
 const supportedGaslessChains = ref([])
+const isMetaMaskInstalled = ref(window?.ethereum?.isMetaMask)
```

Comment:
Don't need ref, since it's not mutated

Reply:
[done](https://github.com/arcana-network/developer-dashboard/pull/368/commits/8510c85eef32a26047f22f25703a1cee05220e05)